### DISCUSSION TITLE
New APIM developer portal content backup

### DISCUSSION THREAD

**Discussion by LaurenceSime** [Azure/api-management-developer-portal#2640] [General]
Hi,

When an APIM instance is backed up, the content of the developer portal is not included in the backup. Currently, as far as I can see, the only way to backup and restore the developer portal content is by running the migrate.js script (https://learn.microsoft.com/en-gb/azure/api-management/automate-portal-deployments). Would it be possible to include a backup and a restore end points to the APIM Management API to backup and restore all content items in a singlerequest?

Thanks.

**Comment by ghost** [comment]
@LaurenceSime, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ygrik** [comment]
@mikebudzynski, can you please comment do we have plans to work on this?
